RFS/docs/DESIGN.md

```diff
@@ -142,13 +134,13 @@ The unit of work for an RFS Worker migrating Elasticsearch Documents is an Elast
 
 ## How the RFS Worker works
 
-In this section, we describe in a high-level, narrative manner how the RFS Worker operates. Detailed state machine diagrams that outline the behavior more explicitly are below as well (see [Appendix: RFS Worker State Machine](#appendix-rfs-worker-state-machine)).  The state machine diagram is intended to be the source of truth, so favor its representation over the narrative description below if conflicts exist.
+In this section, we describe in a high-level, narrative manner how the RFS Worker operates.
 
 ### RFS Worker threads
 
 The RFS Worker’s running process has at least two running threads:
 
-* Main Thread - performs the work of moving the data/metadata from the source cluster to the target cluster; starts the Metrics and Healthcheck Threads
+* Main Thread - performs the work of moving the data from the source cluster to the target cluster; starts the Metrics and Healthcheck Threads
```

Comment:
There likely is a thread for metrics, but do you care to mention it?  There's also more than 1 thread responsible for moving data to the target now that we're using more work fanouts from reactor.

Reply:
Just missed that reference to the Metrics thread; thanks for catching.

IIRC from convos w/ @mikaylathompson, we're going to be move from multiple threads for reindexing to a single thread in order for the checkpointing to work as intended.  We'll just need to make sure that we update the doc here for the final behavior.